### DISCUSSION TITLE
Incorrect file names in Russian when unpacking an archive using 7Zip

### DISCUSSION THREAD

**Discussion by Dima3108** [icsharpcode/SharpZipLib#567] [Q&A]
### Steps to reproduce
1.Compress files in a folder using example [FastZip](https://github.com/icsharpcode/SharpZipLib/wiki/FastZip)
2.Unpack the resulting zip folder using the [7-Zip](https://www.7-zip.org/) program
3.Unpack the same archive using [FastZip](https://github.com/icsharpcode/SharpZipLib/wiki/FastZip) unpacking example

### Expected behavior
Same file and folder names regardless of unpacking programs

### Actual behavior
If the names are in English, there are no problems, but when the files have names in Russian, then 7Zip writes them incorrectly, unlike SharpZipLib. Also, if you unpack an archive with names in Russian using SharpZipLib, they will also be displayed incorrectly. This does not happen with the names  of files in English.

### Version of SharpZipLib
1.3.1
### Obtained from (only keep the relevant lines)

- Downloaded from GitHub

**Comment by piksel** [comment]
If you set 
```cs
ZipStrings.UseUnicode = true;
```
before compressing the archive, the file names should be displayed the same way in most zip programs.
